Makefile

```diff
@@ -53,7 +52,7 @@ k8sprovider_debug::
 	(cd provider && CGO_ENABLED=0 go build -o $(WORKING_DIR)/bin/${PROVIDER} -gcflags="all=-N -l" -ldflags "-X ${PROJECT}/${VERSION_PATH}=${VERSION}" $(PROJECT)/${PROVIDER_PATH}/cmd/$(PROVIDER))
 
 test_provider::
-	cd provider/pkg && go test -short -v -count=1 -coverprofile="coverage.txt" -coverpkg=./... -timeout 2h -parallel ${TESTPARALLELISM} ./...
+	cd provider/pkg && go test -short -v -coverprofile="coverage.txt" -coverpkg=./... -timeout 2h ./...
```

Comment:
Unfortunately, Ginkgo tests don't support the `--parallel` flag when run via `go test`.  However, the `ginkgo` CLI is a drop-in replacement that does.  The overall execution time for `cd provider/pkg && go test` is less than a minute, so the parallelism doesn't seem too important. For that reason I feel we can avoid the `ginkgo` CLI for now.